dist/trip.js

```diff
@@ -1056,6 +1056,26 @@
         'e s w n screen-ne screen-se screen-sw screen-nw screen-center');
       $tripBlock.addClass(o.position);
 
+      // if we have a nextClickSelector use that as the trigger for the next button
+      if(o.nextClickSelector){
+        var eventName = (o.nextClickEvent || 'click')+'.Trip'
```

Comment:
Why you need `o.nextClickEvent` here ? I think normally we only need `click.Trip` and there is no much benefit here for developers to customize their own event here. This will introduce one more configuration which is rare to be used. I prefer to use 'click.Trip' and it's enough !